packages/gatsby-transformer-sharp/src/extend-node-type.js

```diff
@@ -12,6 +13,16 @@ const {
   responsiveResolution,
 } = require(`gatsby-plugin-sharp`)
 
+const ImageFormatType = new GraphQLEnumType({
+  name: `ImageFormat`,
+  values: {
+    default: { value: `` },
```

Comment:
Oh also typically the enum keys are upper case e.g. http://graphql.org/graphql-js/type/#example-4